templates/CRM/common/success.tpl

```diff
@@ -12,42 +12,73 @@
     <h2>{$pageTitle}</h2>
 {/if}
 {if !$upgraded}
-  <div class="crm-container" style="margin-top: 2em; padding: 1em; background-color: #FFFFE3; border: 1px #F8FF00 solid; color: black;">
-    <form method="post">
-        <p>
-          <span class="crm-status-icon info"> </span>
-          {ts 1=$currentVersion 2=$newVersion}The database will be upgraded from %1 to %2.{/ts}
-        </p>
-        {if $preUpgradeMessage}
-            <div style="border: 2px solid #E43D2B; background-color: rgba(228, 61, 43, 0.08); padding: 10px; margin-bottom: 15px;">
-              <span class="crm-status-icon"></span>
-              <strong style="vertical-align: middle; font-size: 1.2em;">{ts}Warning:{/ts}</strong>
-              {$preUpgradeMessage}
-            </div>
+  <div class="crm-container" style="background-color: #FFFFE3; border: 1px #F8FF00 solid;">
+    <div class="crm-success-wrapper">
+      <form method="post">
+        {if $preUpgradeMessageX}
```

Comment:
X?

Reply:
xx

Reply:
fixed :see_no_evil: 
that was me testing what happens when there are no upgrade messages ;)